homeassistant/components/lg_thinq/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/lg_thinq",
   "iot_class": "cloud_push",
   "loggers": ["thinqconnect"],
-  "requirements": ["thinqconnect==1.0.4"]
+  "requirements": ["thinqconnect==1.0.5"]
```

Comment:
Please split the dependency bump from adding a new feature into separate pull requests.

Thanks 👍 

../Frenck